src/jobs/release-lerna-packages.yml

```diff
@@ -46,7 +46,9 @@ steps:
   - configure-github
   - restore_cache:
       keys:
+        - v3-npm-{{ CIRCLE_BRANCH }}-{{checksum "yarn.lock"}}
         - v3-npm-{{checksum "yarn.lock"}}
+        - v3-npm
```

Comment:
@amphro @RodEsp

Reply:
👍

Reply:
Can you also change the `- save_cache:` on line 59 to include the branch?